src/main/java/com/jalasoft/sdfc/pages/Assets/AssetPage.java

```diff
@@ -0,0 +1,43 @@
+package com.jalasoft.sdfc.pages.Assets;/*
+ * @autor: mauricioramirez
+ * Project: sdfc
+ * Package: com.jalasoft.sdfc.pages.Assets
+ * Version: 1.0
+ */
+
+import com.jalasoft.sdfc.core.ui.ISteps;
+import com.jalasoft.sdfc.pages.AbstractPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AssetPage extends AbstractPage {
+
+    @FindBy(css = "nav.entityNameTitle + div.slds-page-header__title > span")
+    private WebElement pageTitle;
+
+    @FindBy(xpath = "//descendant::li[contains(@class, 'tabs__item')] /" +
+            "descendant::span[@class='title' and text()='%s'] / parent::a")
+    private WebElement tabHeaderLink;
```

Comment:
remove this if will never be used, and as far as I know that xpath locator should be in a private String in order to be completed through String.format()